.storybook/preview.js

```diff
@@ -0,0 +1,19 @@
+// export const parameters = {
```

Comment:
Remove commented code? Or is this related to the upcoming plugin work?

Reply:
nope - goood catch ty - updated @demariadaniel